AnimatedWordCloud/Utils/Vector.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2023 AnimatedWordCloud Project
+# https://github.com/konbraphat51/AnimatedWordCloud
+#
+# Licensed under the MIT License.
+"""
+Modules for 2D vector calculation
+"""
+
+from __future__ import annotations
+
+
+class Vector:
+    """
+    Class for 2D vector calculation
+    """
+
+    def __init__(self, *args: float | tuple[float, float]) -> None:
+        """
+        Initialize Vector class.
+
+        :param args: x and y or tuple of x and y
+        input as Vector(x, y) or Vector((x, y))
+        """
+        if args[0].__class__ == tuple:
+            self.x = args[0][0]
```

Comment:
Ensure len(args[0]) == 2

Reply:
or Ensure len(args[0]) >= 2

Reply:
I don't know how best it is to fix it.
Maybe make sure that args[0][0] is a NUMBER and args[0][1] is a NUMBER

Reply:
True,
I gave really better error handling